recipes/check_shapes/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "check_shapes" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b699fcb1e60bb17e2c97007e444b89eeeea2a9102ff11d61fb52454af5348403
+
+build:
+  number: 0
+
+outputs:
+  - name: check_shapes
+    build:
+      noarch: python
+      script_env:
+        # Workaround an upstream conda-build issue w/pip & `outputs` by setting env vars manually.
+        # xref: https://github.com/conda/conda-build/issues/3993
+        - PIP_NO_BUILD_ISOLATION=False
+        - PIP_NO_DEPENDENCIES=True
+        - PIP_IGNORE_INSTALLED=True
+        - PIP_CACHE_DIR=pip_cache
+        - PIP_NO_INDEX=True
+      script: python -m pip install . -vv
```

I believe all that can be:

```suggestion
      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

That is what we are using in all multiple outputs recipes BTW.